zeebe/exporters/opensearch-exporter/src/test/java/io/camunda/zeebe/exporter/opensearch/TestClient.java

```diff
@@ -165,6 +167,36 @@ private PutIndexStateManagementPolicyRequest createPutIndexManagementPolicyReque
     return new PutIndexStateManagementPolicyRequest(policy);
   }
 
+  public Optional<IndexISMPolicyDto> explainIndex(final String index) {
+    try {
+      final var request = new Request("GET", "_plugins/_ism/explain/" + index);
+      final var response = restClient.performRequest(request);
+      final TypeReference<Map<String, Object>> mapTypeReference = new TypeReference<>() {};
+      final Map<String, Object> output =
+          MAPPER.readValue(response.getEntity().getContent(), mapTypeReference);
+      final var policy =
```

Comment:
nice trick :)

Reply:
Thanks! I wished I could have done it in a cleaner way though 🙏